utils/quay_api.py

```diff
@@ -193,3 +186,21 @@ def _repo_change_visibility(self, repo_name, visibility):
         r = requests.post(url, json=params, headers=self.auth_header)
         if not r.ok:
             raise RequestsException(r)
+
+    def get_repo_team_permissions(self, repo_name, team_name):
+        url = f"{self.API_URL}/repository/{self.organization}/" +\
+              f"{repo_name}/permissions/team/{team_name}"
+        r = requests.get(url, headers=self.auth_header)
+        if not r.ok:
+            # the API returns 400 when there is no access
+            return None
+
+        return r.json().get('role') or None
+
+    def set_repo_team_permissions(self, repo_name, team_name, role):
```

Comment:
nit: in some places in this class you use `team` and in other `team_name` and it is a bit confusing 😅

Reply:
changed to `team`